src/sinol_make/commands/export/__init__.py

```diff
@@ -0,0 +1,287 @@
+            tar.add(ocen_dir, arcname=os.path.basename(ocen_dir))
+
+        print("Creating debian package...")
+        os.makedirs(os.path.join(deb_dir, 'usr', 'share', f'oi-ocen-{self.task_id}'), exist_ok=True)
+        shutil.copyfile(os.path.join(tmpdir, 'ocen-linux.tgz'),
+                        os.path.join(deb_dir, 'usr', 'share', f'oi-ocen-{self.task_id}', 'ocen-linux.tgz'))
+
+        def replace_text(file, old, new):
+            with open(file, 'r') as f:
+                content = f.read().replace(old, new)
+            with open(file, 'w') as f:
+                f.write(content)
+
+        replace_text(os.path.join(deb_dir, 'DEBIAN', 'control'), 'TASK_ID', self.task_id)
+        replace_text(os.path.join(deb_dir, 'DEBIAN', 'postinst'), 'task_id = None', f'task_id = "{self.task_id}"')
+
+        return self.build_debian_package(tmpdir, f'oi-ocen-{self.task_id}')
+
+    def make_ocen(self, tmpdir):
+        """
+        Creates ocen debian package for contests.
+        :param tmpdir: Temporary directory path.
+        :return: Path to ocen package.
+        """
+
+        print("Copying ocen template...")
+        ocen_template = os.path.abspath(os.path.join(util.get_templates_dir(), 'ocen-linux-sio2jail'))
+        ocen_dir = os.path.join(tmpdir, 'rozw')
+        shutil.copytree(ocen_template, ocen_dir)
+        for d in ['in', 'out']:
+            shutil.rmtree(os.path.join(ocen_dir, d))
+            os.mkdir(os.path.join(ocen_dir, d), 0o755)
+
+        for root, dirs, files in os.walk(ocen_dir):
+            for f in files:
+                util.fix_line_endings(os.path.join(root, f))
+
+        test_patterns = [
+            'in/???0*.in',
+            'in/*ocen.in',
+            'out/???0*.out',
+            'out/*ocen.out',
+        ]
+
+        nums = set()
+        for pattern in test_patterns:
+            for file in glob.glob(os.path.join(os.getcwd(), pattern)):
+                self.copy_test(ocen_dir, file)
+                nums.add(os.path.basename(file).split('.')[0][3:])
+
+        self.ocen_gen_conf(ocen_dir, nums)
+        return self.ocen_make_deb_package(tmpdir, ocen_dir)
+
+    def make_dlazaw(self, tmpdir):
+        """
+        Creates dlazaw debian package for contests.
+        :param tmpdir: Temporary directory path.
+        """
+
+        print("Copying dlazaw template...")
+        shutil.copytree(os.path.join(util.get_templates_dir(), 'oi-dlazaw'),
+                        os.path.join(tmpdir, 'oi-dlazaw'))
+
+        shutil.copytree(os.path.join(os.getcwd(), 'dlazaw'),
+                        os.path.join(tmpdir, 'oi-dlazaw', 'home', 'zawodnik', 'dlazaw'))
+
+        print("Creating debian package...")
+        return self.build_debian_package(tmpdir, 'oi-dlazaw')
+
+    def run(self, args: argparse.Namespace):
+        if not util.check_if_project():
+            print(util.warning('You are not in a project directory (couldn\'t find config.yml in current directory).'))
+            exit(1)
```

Comment:
Is this code copy-pasted from other commands? It seems reasonable to create a helper function for that.

Reply:
I added a helper function like that